x/valset-pref/types/expected_interfaces.go

```diff
@@ -19,3 +19,10 @@ type StakingInterface interface {
 type BankKeeper interface {
 	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
 }
+
+type CommunityPoolKeeper interface {
```

Comment:
```suggestion
type DistributionKeeper interface {
```

Reply:
Why for testing only?

Reply:
Its only used in testing? The main logic shouldn't use it?

Reply:
makes sense

Reply:
We are using `WithdrawDelegationRewards(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) (sdk.Coins, error)` in code that falls under this interface